pkg/scheduler/util/backoff_utils.go

```diff
@@ -142,26 +142,6 @@ func (p *PodBackoff) BackoffPod(podID ktypes.NamespacedName) time.Duration {
 	return entry.getBackoff(p.maxDuration)
 }
 
-// TryBackoffAndWait tries to acquire the backoff lock
```

Comment:
Is there anything in this file that is still used? We should check and if possible, remove the whole file. Our new backoff util resides in scheduler/internal/queue